# Our MaterialGroup message. Contains image index, default color and array of face indices.
Header header
sensor_msgs/Image image
std_msgs/ColorRGBA defaultcolor
uint32[] face_indices
